### DISCUSSION TITLE
不如讨论下更好的传播信息

### DISCUSSION THREAD

**Discussion by C-Joey** [WeNeedHome/SummaryOfLoanSuspension#637] [General]
比如：
1.给别人链接的时候并且告知github repo链接的时候再给一个加速站，能够顺畅访问
2.不要在国内社交软件聊天群里直接发链接
。。。

**Comment by Crazy-Creeper** [comment]
传的越快死的越快

**Comment by liuwenwu520** [comment]
我觉得应该讨论的是在国内有哪些可平台让业主发声，让更多的人看到各处烂尾楼的情况

**Reply by taopeng-life** [reply]
没有一个平台可以发声，和谐的很

**Reply by Crazy-Creeper** [reply]
GJ怎么会变成这个样子，只会嘴巴一堵耳朵一捂。

**Comment by uueer** [comment]
哈哈哈  小心你微博号都没

**Reply by C-Joey** [reply]
一年前就没了 四个账号
